services/onboarding_service.rb

```diff
@@ -0,0 +1,82 @@
+require "json"
+require_relative "../shared/logging_module"
+
+module FastlaneCI
+  # Provides operations to create and mutate the FastlaneCI configuration
+  # repository
+  class OnboardingService
+    include FastlaneCI::Logging
+
+    # Spawns a process that opens the onboarding page using the user's preferred
+    # web browser
+    def launch_onboarding!
+      logger.info("Launching onboarding web page.")
+      `open http://localhost:8080/onboarding`
```

Comment:
TODO: I need to change this url. 😳

Reply:
Something that Jenkins do is to provide in the logs (for the first time) a link url with the password hash in order to authenticate the first user. Something like:
```
_fastlane.ci_ has launched successfully, visit URL in order to finish the onboarding process
```
Also, I'll love to see in the page that is opened:
`I have an existing ci-repo`, `I don't have an existing ci-repo`